Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,175 @@
+<br>
+<div class=“alert alert-info” role=“alert”>
+<i class=“fa fa-exclamation-triangle”></i><b> Note:</b> This documentation is for an alpha feature. For questions and feedback on the Multi-cluster Alpha program, email <a href="mailto:tectonic-alpha-feedback@coreos.com">tectonic-alpha-feedback@coreos.com</a>.
+</div>
+
+# Installing the multi-cluster registry
+
+Use Tectonic's multi-cluster registry to:
+* maintain a list of available clusters
+* enable cluster administrators to configure base security policies, and enforce other standardization policies across multiple clusters
+
+Use this guide to configure the registry.
+
+## Set up the directory cluster
+
+Select one of your clusters to be the "directory cluster", which is the cluster that is the source of truth for your policies. Other clusters will be configured to connect to this one. Check that kubectl is set up correctly and can talk to the cluster:
+
+```
+export KUBECONFIG=/path/to/kubeconfig
+kubectl get nodes
+```
+
+Next, read back the list of Clusters. There shouldn’t be any yet:
+
+```
+$ kubectl get clusters
+No resources found.
+```
+
+Create a new Cluster object that represents this cluster. Make sure this matches the name you installed with:
+
+```
+kubectl --namespace=tectonic-system get configmap/tectonic-config -o jsonpath="{..data.clusterName}"
+west-coast-production
+```
+
+Give the cluster a set of labels. These labels will be used in Cluster Policies:
+
+```
+apiVersion: multicluster.coreos.com/v1alpha1
```

Comment:
This is

```
multicluster.coreos.com/v1
```

Reply:
question: in multi-cluster-policy.md the line is:

apiVersion: "multicluster.coreos.com/v1"

which is correct? quotes? or no quotes?

Reply:
it doesn't matter actually